core/src/main/java/org/testcontainers/utility/TestcontainersConfiguration.java

```diff
@@ -65,7 +64,6 @@
         .put(DockerImageName.parse(SOCAT_IMAGE), "socat.container.image")
         .put(DockerImageName.parse(VNC_RECORDER_IMAGE), "vncrecorder.container.image")
         .put(DockerImageName.parse(COMPOSE_IMAGE), "compose.container.image")
-        .put(DockerImageName.parse(ALPINE_IMAGE), "tinyimage.container.image")
```

Comment:
this isn't fully backward compatible... WDYT about defaulting to Ryuk, but also letting users override it?

Reply:
It would be really problematic wrt. image substitution...

All the other image substitutions work bidirectionally: the user is e.g. asserting that `compose.container.image` should, if set, point to a suitable substitute for `docker/compose`. When Testcontainers sees a code reference to `docker/compose`, it's OK to replace it with the provided image.

Tiny image == Alpine means that any usage of `alpine` will be substituted by whatever the tiny image override points to. It's kinda like saying Tiny image == Alpine, therefore Alpine == Tiny image

If we set the code - this Map - we'd be saying Tiny image == Ryuk, therefore Ryuk == Tiny image. That will break (badly) if anybody ever configures tiny image to point to something that is not a suitable substitute for Ryuk.

This is why I've removed this line: with this PR `tinyimage.container.image` is (a) useless, and (b) potentially totally breaking if it is used.

If this setting had been called `alpine.container.image` then sadly everything would be fine 😭 😂 

Honestly I'm half tempted to rip out the entire `*.container.image` configuration thing and guide people to use the full image name substitution system instead, but that's too big of a change, really 😆

Reply:
> If we set the code - this Map - we'd be saying Tiny image == Ryuk, therefore Ryuk == Tiny image. That will break (badly) if anybody ever configures tiny image to point to something that is not a suitable substitute for Ryuk.

We don't have to associate one to another. Rather "use Ryuk's image as the default for `tinyimage.container.image`, and use `tinyimage.container.image` as an image for checks".
If somebody sets the image, Ryuk will use Ryuk's image, while checks will use the custom provided image.

Reply:
Perhaps we should discuss on a call - it's complicated!